packages/pharos/src/components/coach-mark/pharos-coach-mark.scss

```diff
@@ -0,0 +1,183 @@
+@keyframes fadein {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes fadein-short {
+  0% {
+    opacity: 0;
+  }
+
+  37.5% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes fadein-long {
+  0% {
+    opacity: 0;
+  }
+
+  66.66% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+:host {
+  --coach-mark-background-color: var(--pharos-color-white);
+  --coach-mark-border-color: var(--pharos-color-marble-gray-base);
+  --coach-mark-border-width: 1px;
+  --coach-mark-text-size: var(--pharos-font-size-small);
+  --coach-mark-heading-color: var(--pharos-color-marble-gray-20);
+  --coach-mark-text-color: var(--pharos-color-marble-gray-20);
+  --coach-mark-text-spacing: var(--pharos-spacing-1-x);
+  --coach-mark-pip-size: 10px;
+
+  display: block;
+  position: absolute;
+}
+
+:host([variant='dark']) {
+  --coach-mark-background-color: var(--pharos-button-color-base-overlay-background-base);
+  --coach-mark-border-color: var(--coach-mark-background-color);
+  --coach-mark-heading-color: var(--pharos-color-white);
+  --coach-mark-text-color: var(--pharos-color-marble-gray-80);
+}
```

Comment:
These have typically been defined as component tokens up at the top level with all the other tokens (see `packages/pharos/tokens/components/`). Recently we've been discussing whether to continue using component tokens like that at all, so I'm a little uncertain what to suggest here; if we were to get rid of component tokens at the top level, I like this way of handling the CSS properties. Interested in thoughts from other @ithaka/pharos-maintainers

Reply:
Should I move these values to `packages/pharos/tokens/components/` in the mean time?

Reply:
If it isn't too much trouble, I think that's the right move for now. Be aware of [this decision](https://github.com/ithaka/pharos/blob/develop/docs/architecture/decisions/0008-evolve-tokens-in-backward-compatible-manner.md) that means I think you'll need to provide the more raw values above as fallbacks when using the tokens in this component.

Reply:
I've moved the pharos tokens to `packages/pharos/tokens/components/coach-mark.json`, but I'm using them to populate the light/dark color variables here so I don't need to refactor the color theme switching